app/src/main/java/com/android/sample/ui/composables/Legend.kt

```diff
@@ -53,3 +56,36 @@ fun Legend() {
         LegendItem(LegendRunning, stringResource(R.string.TitleTabRunning))
       }
 }
+
+/** Displays a row of workout type legends but small (e.g., Bodyweight, Yoga, Running). */
+@Composable
+fun SmallLegend() {
+  Row(
+      modifier = Modifier.fillMaxWidth().height(30.dp).testTag("Categories"),
+      horizontalArrangement =
+          Arrangement.spacedBy(space = 20.dp, alignment = Alignment.CenterHorizontally)) {
+        SmallLegendItem(LegendBodyweight, stringResource(R.string.TitleTabBody))
+        SmallLegendItem(LegendYoga, stringResource(R.string.TitleTabYoga))
+        SmallLegendItem(LegendRunning, stringResource(R.string.TitleTabRunning))
+      }
+}
+
+/**
+ * Displays an individual small legend item with a colored background and text.
+ *
+ * @param color The color for the legend item.
+ * @param text The text description for the workout type.
+ */
+@Composable
+fun SmallLegendItem(color: Color, text: String) {
+  val shape =
+      RoundedCornerShape(topStart = 15.dp, topEnd = 5.dp, bottomStart = 5.dp, bottomEnd = 15.dp)
+  Box(
+      modifier =
+          Modifier.testTag("legendItem")
+              .shadow(4.dp, shape = shape)
+              .background(color, shape = shape)
+              .padding(horizontal = 8.dp)) {
+        Text(text = text, color = Color.Black, fontSize = 14.sp, fontFamily = OpenSans)
```

Comment:
What the fuck is Color.Black? I only know black from Color.kt, you should use it!